### DISCUSSION TITLE
Doris 外部表能否减轻数据库压力

### DISCUSSION THREAD

**Discussion by 15374604475** [apache/doris#16611] [RFC]
我们有个实时报表需求：当前业务数据存储在mysql，为了减轻mysql压力，准备写到doris。如果我建个外表，SQL还是会指向mysql吧，压力依旧不减吧？

**Comment by 15374604475** [comment]
还有种方案就是 读取mysql binlog 写到 kafka，然后再sink到kafka

**Reply by LemonLiTree** [reply]
外表是从mysql读数据，计算在doris。比较推荐的方案是flink cdc同步mysql数据到doris，在doris做查询即可。
